content/applications/productivity/iot/config/windows_iot.rst

```diff
@@ -0,0 +1,121 @@
+
+   Using ``C:\odoo`` will allow for the Nginx server to start. Click :guilabel:`Install`.
+
+#. Allow the installation to occur. This may take a few minutes. Click :guilabel:`Next` to continue.
+
+#. Ensure that the :guilabel:`Start Odoo` box is checked and click :guilabel:`Finish`.
+
+#. After installation, the Odoo server will run and automatically open ``localhost:8069`` on your
+   web browser. The webpage should load into the IoT Box Configuration Page.
+
+   .. important::
+      A manual restart of the Odoo server may be necessary should the browser give an error.
+      To restart the virtual Windows IoT server:\
+
+      - Type "Services" into the :guilabel:`Search Bar`
+      - Select the :menuselection:`Services` App and scroll down to the :guilabel:`Odoo` Service.
+      - Right click on :guilabel:`Odoo` and select :guilabel:`Start` or :guilabel:`Restart`. This
+        action will manually restart the Odoo :abbr:`IoT (Internet of Things)`  server.
+
+#. Connect devices to Windows computer.
+
+#. Windows should automatically detect the device because the driver is pre-installed on
+   the computer. If not, search and install the Windows driver for their device.
+
+#. Refresh the :abbr:`IoT (Internet of Things)`  Box Configuration Page and verify the device is
+   seen on the configuration page. If not, reload the handlers through the configuration page.
+
+#. Connect Windows IoT to a database using existing instructions (manually using the Token).
+
+   .. seealso::
+      :doc:`connect`
+
+#. Use the devices connected to :abbr:`IoT (Internet of Things)` as you normally would.
+
+
+Troubleshooting
+===============
+
+Virtual Windows IoT box not showing on the database
+---------------------------------------------------
+
+In some instances a manual restart of the physical :abbr:`IoT (Internet of Things)` box can resolve
+the issue. For the Windows virtual :abbr:`IoT (Internet of Things)` box a manual restart of the Odoo
+server can resolve database connection issues.
+
+To restart the virtual Windows IoT server:
+
+#. Type "Services" into the :guilabel:`Search Bar`
+#. Select the :menuselection:`Services` App and scroll down to the :guilabel:`Odoo` Service.
+#. Right click on :guilabel:`Odoo` and select :guilabel:`Start`. This action will manually restart
+   the Odoo IoT server.
+
+Windows Firewall
+----------------
+
+The Windows virtual :abbr:`IoT (Internet of Things)` box software may not connect to the :abbr:`LAN
+(Local Area Network)` due to the Windows Firewall preventing the connection. Consult your local IT
+support to make exceptions in the :abbr:`OS (Operating System)`.
+
+
+Uninstalling Windows IoT
+------------------------
+
+Uninstalling the Windows virtual IoT is done through the Windows program manager. Search in any
+Windows version for ''program''. Select :guilabel:`Add or Remove Programs` located in the control
+panel. Search for ``Odoo`` and click the :guilabel:`three dot menu` to uninstall.
```

Comment:
Rephrase; also, just searching "Odoo" may bring up multiple programs, so you should specify the exact name of the program being uninstalled
```suggestion
Uninstalling the Windows virtual IoT box is done through the Windows program manager. From the
:guilabel:`Start` menu, search for ''program''. Select :guilabel:`Add or Remove Programs` . In the window that appears, search for ``Odoo``, click the :guilabel:`three dot menu`, then select :guilabel:`Uninstall`.
```

Reply:
Odoo is the name of the program that should be uninstalled. I tested this a couple times. What did you encounter?

Reply:
Did you encounter multiple Odoo programs? The uninstall should be from Odoo and there are other programs that will be uninstalled.